### PR TITLE
Pants& professions

### PR DESCRIPTION
**Pull request by The-Anjelo-Guy**
#### Summary
Content "I think I did it properly now.  Attempted to fix the mess with pants in the professions, selecting roper variants rather than a random one.  Gave the handyman/handywoman professions a generic tank top, instead of a camo or cheerleader one, added some female names for other professions such as the Lumberjack (Lumberjill), fixed issues with the holy symbol regarding some professions, selecting a proper variant, added a new profession (Asatru priest or priestess), and a new religious book, the Asatru Edda."

#### Purpose of change
I found it very weird that my handyman would start wearing a cheerleader tank top and pink pants.

### CODE REVIEW

File: data/json/items/book/spiritual.json [hunk 1]
```diff
@@ -654,5 +654,14 @@
     "weight": "292 g",
     "volume": "750 ml",
     "copy-from": "holybook_abstract"
+  },
+  {
+    "id": "holybook_asatru",
+    "type": "BOOK",
+    "name": { "str": "The Asatru Edda: Sacred Lore of the North", "str_pl": "copies of The Asatru Edda: Sacred Lore of the North" },
+    "description": "A commercial-grade, hardcover English translation of The Asatru Edda.  It serves as a modern guide to the religious and mythological beliefs of the followers of Asatru, a contemporary revival of Norse paganism.  This book contains a collection of poems, stories, rituals, and information about the gods, goddesses, heroes, and cosmology of the Norse tradition, from ancient writings and oral traditions.",
```
Comment (harakka) [inline data/json/items/book/spiritual.json:662]: There is no "The Asatru Edda" for there to be an English translation of, and there's no holy book in Asatru, despite someone apparently coming up with one and selling it on Amazon. If someone considers this book holy they're a deep minority and not representative of anything.

**Comment by RenechCDDA** [comment]
There are so few practitioners in the US I do not believe this can be added.

Or to be more specific, here's an expert citation which you can verify:

["It is impossible to posit with certainty the precise size of the American Asatru community [...] 500 committed adherents would be a reasonable estimate."](https://www.researchgate.net/profile/Jeffrey-Kaplan/publication/329415637_The_Reconstruction_of_the_Asatru_and_Odinist_Traditions/links/5c0783a1a6fdcc315f9dec62/The-Reconstruction-of-the-Asatru-and-Odinist-Traditions.pdf)

**Comment by The-Anjelo-Guy** [comment]
> There are so few practitioners in the US I do not believe this can be added.
> 
> Or to be more specific, here's an expert citation which you can verify:
> 
> ["It is impossible to posit with certainty the precise size of the American Asatru community [...] 500 committed adherents would be a reasonable estimate."](https://www.researchgate.net/profile/Jeffrey-Kaplan/publication/329415637_The_Reconstruction_of_the_Asatru_and_Odinist_Traditions/links/5c0783a1a6fdcc315f9dec62/The-Reconstruction-of-the-Asatru-and-Odinist-Traditions.pdf)

There's a literal Necronomicon in the game but yeah. Also, there is the Hávámál, and lots of other stuff that technically are not common or just very rare in the New England region.

File: data/json/items/book/spiritual.json (hunk 1 above)
Comment (The-Anjelo-Guy) [reply data/json/items/book/spiritual.json:662]: If you say so, that's a valid reason to erase this item and use the Hávámál instead.

**Comment by The-Anjelo-Guy** [comment]
> There are so few practitioners in the US I do not believe this can be added.
> 
> Or to be more specific, here's an expert citation which you can verify:
> 
> ["It is impossible to posit with certainty the precise size of the American Asatru community [...] 500 committed adherents would be a reasonable estimate."](https://www.researchgate.net/profile/Jeffrey-Kaplan/publication/329415637_The_Reconstruction_of_the_Asatru_and_Odinist_Traditions/links/5c0783a1a6fdcc315f9dec62/The-Reconstruction-of-the-Asatru-and-Odinist-Traditions.pdf)

That is really unfortunate, I was hoping for this book to get approved. Thank you so much for providing me with this valuable information on the matter.

**Pull request merged**